docs/content/sips/005-manifest-redesign.md

```diff
@@ -61,31 +85,41 @@ In order to decouple triggers and components, we move triggers to new top-level
 ```toml
 [[trigger.http]]
 route = "/world"
-component = "world"
-
-[[component]]
-id = "world"
+handler = "world"
+[component.world]
 # ...
 ```
 
-### Manifest version
+### Component inline config
 
-Since this proposal represents a backward-incompatible change to the manifest format, the manifest
-version changes. The existing `spin_version = "1"` field name leaves some room for misinterpretation
-as "compatible with Spin project version 1". As an optional part of this proposal, change that name:
+In addition to being defined in their own top-level sections and referenced by ID,
+component configs may be inlined; allowing this:
```

Comment:
Why does the proposal want to allow this?

Reply:
It keeps the manifest smaller and linked component/trigger configs visually close together.